### PR TITLE
Entities Configure search configurations matching name

### PR DESCRIPTION
**Pull request by benbosman**
This is a rename of the search configurations to match the entity type. This is relevant for https://github.com/DSpace/dspace-angular/pull/545 because the UI will be easier to work with if there's a search configuration matching the entity type

### CODE REVIEW

**Review by paulo-graca — APPROVED** [review]
Thank you @benbosman ! No comments.

File: dspace/config/spring/api/discovery.xml [hunk 1]
```diff
@@ -57,10 +57,10 @@
...
-                <entry key="publicationIssue" value-ref="publicationIssue"/>
-                <entry key="publicationVolume" value-ref="publicationVolume"/>
-                <entry key="periodical" value-ref="periodical"/>
+                <entry key="orgunit" value-ref="orgUnit"/>
+                <entry key="journalissue" value-ref="journalIssue"/>
+                <entry key="journalvolume" value-ref="journalVolume"/>
+                <entry key="journal" value-ref="journal"/>
                 <entry key="project" value-ref="project"/>
                 <!-- search for an entity that can be a Person or an OrgUnit -->
                 <entry key="personOrOrganization" value-ref="personOrOrganization"/>
```
Comment (paulo-graca) [inline dspace/config/spring/api/discovery.xml]: @benbosman, for coherence, do you also propose to rename this setting (and the references for it) from:
`<entry key="personOrOrganization" value-ref="personOrOrganization"/>`
to 
`<entry key="personOrOrgUnit" value-ref="personOrOrgUnit"/>`

Comment (benbosman) [reply dspace/config/spring/api/discovery.xml]: I've updated this in the PR

**Comment by tdonohue** [comment]
Closed & reopened this PR to trigger Travis to rebuild.  It had not built this in some time & hadn't done a recent analysis.

**Review by tdonohue — APPROVED** [review]
👍 Looks good to me. This is just correcting configurations

**Pull request merged**